.goreleaser.yml

```diff
@@ -3,9 +3,7 @@ builds:
       - CGO_ENABLED=0
       - GOPRIVATE="github.com/spacelift-io"
     mod_timestamp: '{{ .CommitTimestamp }}'
-    flags:
-      - -trimpath
-    ldflags: ['-s -w']
```

Comment:
ldflags removed because of what reason?

Reply:
Because goreleaser provides a richer set by default, and I'm using those defaults.

Reply:
As per docs:

```
    # Default is `-s -w -X main.version={{.Version}} -X main.commit={{.Commit}} -X main.date={{.Date}} -X main.builtBy=goreleaser`.
```

Reply:
OK